plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -954,6 +966,24 @@ def try_get_gas_pricing(
             gas_price = self._api.eth.generate_gas_price()
         finally:
             self._api.eth.set_gas_price_strategy(prior_strategy)  # pragma: nocover
+
+        if old_tip is not None and gas_price_strategy is EIP1559:
+            base_fee_per_gas = (
+                gas_price["maxFeePerGas"] - gas_price["maxPriorityFeePerGas"]
+            )
+            updated_max_priority_fee_per_gas = old_tip * TIP_INCREASE
+            updated_max_fee_per_gas = (
+                updated_max_priority_fee_per_gas + base_fee_per_gas
+            )
+
+            if gas_price["maxPriorityFeePerGas"] < updated_max_priority_fee_per_gas:
+                gas_price["maxPriorityFeePerGas"] = updated_max_priority_fee_per_gas
+                gas_price["maxFeePerGas"] = updated_max_fee_per_gas
+
+        elif old_tip is not None and gas_price_strategy is GAS_STATION:
+            updated_gas_price = old_tip * TIP_INCREASE
```

Comment:
In the old pricing strategy there was only `gasPrice`, there was no concept of a tip.